assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -254,23 +254,28 @@ spec:
         - containerPort: 8000
           name: http
           protocol: TCP
-        resources: {}
+{{ if .PortierisContainerResources }}
+        resources:{{ range .PortierisContainerResources }}{{ range .ResourceRequest }}
+          requests: {{ if .CPU }}
+            cpu: {{ .CPU }}{{ end }}{{ if .Memory }}
+            memory: {{ .Memory }}{{ end }}{{ end }}{{ range .ResourceLimit }}
+          limits: {{ if .CPU }}
+            cpu: {{ .CPU }}{{ end }}{{ if .Memory }}
+            memory: {{ .Memory }}{{ end }}{{ end }}{{ end }}
+{{ end }}
+{{ if .KubeAPIServerSecurity }}
```

Comment:
Please leave the security context work to https://github.com/openshift/ibm-roks-toolkit/pull/114.